easybuild/easyconfigs/p/PCAngsd/PCAngsd-python-hashbang.patch

```diff
@@ -0,0 +1,8 @@
+diff -ru pcangsd-0.97/pcangsd.py pcangsd-0.97.patched/pcangsd.py
```

Comment:
@FokkeDijkstra Please include a short description of the patch above this line + mention author (and remember to update the SHA256 checksum for the patch accordingly)

Reply:
Done.